features/draft_environment.feature

```diff
@@ -11,14 +11,14 @@ Feature: Draft environment
     Then I should be on the case study page
     And the page should contain the draft watermark
 
-  @draft
+  @draft @high
```

Comment:
I would just remove the Nagios check for this. It's valuable to know that Smokey has failed, but we shouldn't be notifying the alerting system.

To make this check more useful for Smokey we should also do what you suggest with content-register.